finality-aleph/src/new_network/connection_manager/discovery.rs

```diff
@@ -0,0 +1,690 @@
+use crate::{
+    new_network::{
+        connection_manager::{Authentication, Multiaddr, SessionHandler},
+        DataCommand, PeerId, Protocol,
+    },
+    NodeIndex, SessionId,
+};
+use codec::{Decode, Encode};
+use std::{
+    collections::{HashMap, HashSet},
+    time,
+};
+
+/// Messages used for discovery and authentication.
+#[derive(Clone, Debug, Encode, Decode)]
+pub enum DiscoveryMessage {
+    // Contains the id of the broadcast, to avoid spam.
```

Comment:
I have problems with understanding the term 'id of the broadcast'

Reply:
Uhhh, that was from a former version of spam avoidance, outdated comment removed. ">_>